contracts/PensionFundRelease.sol

```diff
@@ -39,12 +44,14 @@ contract PensionFundRelease {
         require(_firstPaymentPercent <= 100);
         require(_reccurentPaymentPercent <= 100);
 
+        totalReleased = 0;
```

Comment:
Let's have all variables initialized in one place: inside constructor or in declaration, what you like more